functions.R

```diff
@@ -59,6 +59,46 @@ get_survey_data <- function() {
         colnames(temp_dat) <- attr(temp_dat,"variable.labels")
         # get the column names we want from are varibale list
         temp_sub <- clean_subset_survey(temp_dat, get_year = get_year, folder = temp_folder)
+        
+        colnames(temp_sub)[grepl('age|Age', colnames(temp_sub))]
+        summary(as.factor(temp_sub$respondents_age))
+        # remove age groups that are above 29 
```

Comment:
@benmbrew Are there any younger age groups we need to be concerned about (ie, 10-15, etc)?

Reply:
oops, ignore 63 and 64.
But no, I checked all the different levels and got the ones we need. no worries here

Reply:
Remove lines that are irrelevant (rather than "ignore")

Reply:
I'm removing now